Cargo.toml

```diff
@@ -198,8 +198,8 @@ beefy-gadget = { git = "https://github.com/paritytech//grandpa-bridge-gadget", r
 beefy-gadget-rpc = { git = "https://github.com/paritytech//grandpa-bridge-gadget", rev = "f40c0ab7b327e874d5c8d699bfa5d762f1759ee0" }
 
 [patch."https://github.com/open-web3-stack/open-runtime-module-library"]
-orml-currencies = { git = "https://github.com/open-web3-stack//open-runtime-module-library", rev = "5babe0b71cf60538ca23d41231f9558816c52ae9" }
-orml-tokens = { git = "https://github.com/open-web3-stack//open-runtime-module-library", rev = "5babe0b71cf60538ca23d41231f9558816c52ae9" }
-orml-xtokens = { git = "https://github.com/open-web3-stack//open-runtime-module-library", rev = "5babe0b71cf60538ca23d41231f9558816c52ae9" }
-orml-xcm-support = { git = "https://github.com/open-web3-stack//open-runtime-module-library", rev = "5babe0b71cf60538ca23d41231f9558816c52ae9" }
-orml-traits = { git = "https://github.com/open-web3-stack//open-runtime-module-library", rev = "5babe0b71cf60538ca23d41231f9558816c52ae9" }
+orml-currencies = { path = "../../orml/currencies" }
+orml-tokens = { path = "../../orml/tokens" }
+orml-xtokens = { path = "../../orml/xtokens" }
+orml-xcm-support = {  path = "../../orml/xcm-support" }
+orml-traits = { path = "../../orml/traits" }
```

Comment:
```suggestion
orml-currencies = { git = "https://github.com/open-web3-stack//open-runtime-module-library", rev = "5babe0b71cf60538ca23d41231f9558816c52ae9" }
orml-tokens = { git = "https://github.com/open-web3-stack//open-runtime-module-library", rev = "5babe0b71cf60538ca23d41231f9558816c52ae9" }
orml-xtokens = { git = "https://github.com/open-web3-stack//open-runtime-module-library", rev = "5babe0b71cf60538ca23d41231f9558816c52ae9" }
orml-xcm-support = { git = "https://github.com/open-web3-stack//open-runtime-module-library", rev = "5babe0b71cf60538ca23d41231f9558816c52ae9" }
orml-traits = { git = "https://github.com/open-web3-stack//open-runtime-module-library", rev = "5babe0b71cf60538ca23d41231f9558816c52ae9" }
```

Reply:
oops missed that from the acala fork I'm working on